language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/DamlLedgerClientTest.scala

```diff
@@ -10,13 +10,7 @@ import java.util.concurrent.TimeUnit
 import com.daml.ledger.api.v1.command_completion_service.Checkpoint
 import com.daml.ledger.api.v1.ledger_offset.LedgerOffset
```

Comment:
Pardon my ignorance but doesn't this line indicate that we're still using ledger API v1?

Reply:
We have a number of interfaces that remain untouched in v2. We will have to rename them v1->v2 in one swoop